libs/module-to-standalone/shell/src/lib/main-shell.routes.ts

```diff
@@ -1,36 +1,45 @@
+import { HomeComponent } from './../../../../../apps/anchor-scrolling/src/app/home.component';
 import { IsAuthorizedGuard } from '@angular-challenges/module-to-standalone/admin/shared';
 import { Route } from '@angular/router';
 
 export const appRoutes: Route[] = [
   { path: '', redirectTo: 'home', pathMatch: 'full' },
   {
     path: 'home',
-    loadChildren: () =>
+    loadComponent: () =>
       import('@angular-challenges/module-to-standalone/home').then(
-        (m) => m.ModuleToStandaloneHomeModule
+        (mod) => mod.HomeComponent
```

Comment:
just because I want to say something: `mod` is not the right word here. There is no more module. You should write `c` or `comp`. 

But otherwise that's perfect. Great job 💪

Reply:
I see..
I will change it.